content-warehouse/src/test/java/contentwarehouse/v1/DocumentTests.java

```diff
@@ -87,6 +87,14 @@ public void testUpdateDocument()
     String got = bout.toString();
     assertThat(got).contains("Document");
   }
+
+  @Test
+  public void testSearchDocument()
+      throws InterruptedException, ExecutionException, IOException, TimeoutException {
+    SearchDocuments.searchDocuments(PROJECT_ID, LOCATION, "document", USER_ID);
+    String got = bout.toString();
+    assertThat(got).contains("Document");
```

Comment:
Can this be updated to check for a specific document name?

Reply:
This sample is to search all documents for a specific query. All documents in the test project contain the word 'document' as plain text so I used that here. Another sample will cover searching document names and histogram queries